pkgs/shells/zsh/zsh-fzf-history-search/default.nix

```
@@ -0,0 +1,36 @@
+{ lib
+, stdenvNoCC
+, fetchFromGitHub
+, substituteAll
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "zsh-fzf-history-search";
+  version = "unstable-2023-03-08";
+
+  src = fetchFromGitHub {
+    owner = "joshskidmore";
+    repo = "zsh-fzf-history-search";
+    rev = "d1aae98ccd6ce153c97a5401d79fd36418cd2958";
+    hash = "sha256-4Dp2ehZLO83NhdBOKV0BhYFIvieaZPqiZZZtxsXWRaQ=";
+  };
+
+  strictDeps = true;
+
+  installPhase = ''
+    runHook preInstall
+
+    install -D zsh-fzf-history-search.zsh  --target-directory=$out/share/zsh-fzf-history-search
+    install -D zsh-fzf-history-search.plugin.zsh  --target-directory=$out/share/zsh-fzf-history-search
```

```suggestion
    install -D zsh-fzf-history-search{,.plugin}.zsh  --target-directory=$out/share/zsh-fzf-history-search
```
It may be a matter of preference, but I would like to reduce duplication.

Feel free to ignore this,

---

To be honest I find that not too `{,.plugin}` readable, but I replaced it with a wildcard which works just fine.